dev/breeze/src/airflow_breeze/params/shell_params.py

```diff
@@ -597,7 +596,7 @@ def env_variables_for_docker_commands(self) -> dict[str, str]:
         _set_var(_env, "SKIP_SSH_SETUP", self.skip_ssh_setup)
         _set_var(_env, "SQLITE_URL", self.sqlite_url)
         _set_var(_env, "SSH_PORT", None, SSH_PORT)
-        _set_var(_env, "STANDALONE_DAG_PROCESSOR", self.standalone_dag_processor)
+        _set_var(_env, "STANDALONE_DAG_PROCESSOR", "true")
```

Comment:
I think we need to make it conditional - when `--use-airflow-version` is set to < 2.10.* we probably don't want to run with standalone dag processor or at least give a chance to disable it possibly better solution will be to change `--use-standalone-dag-processor` to two way flag `--use-standalone-dag-processor/--no-use-standalone-dag-processor` and default it to `True`

Reply:
Hmm, do we need that flexibility? Seems reasonable that main could be 3.0+ only.

Reply:
Until we release 2.11 - yeah, we use breeze often (few people I know of) to test airflow releases and when they try to reproduce issues with older airflow versions. It's very convenient, because the old breeze version which is part of the 2.x release might or might not work for varios reasons related to dependency changes and other environmental factors, and would be great to keep it working for a while.

Reply:
(BTW - it IS a bit broken now - because it also starts fast-api today when yoy run old version - I wanted to fix it as well though, because it's annoying to see it fail...

Reply:
Well, that was pretty easy: #45723

Reply:
Fix for the fastapi annoyance https://github.com/apache/airflow/pull/45724

Reply:
> Well, that was pretty easy: https://github.com/apache/airflow/pull/45723

Yeah, I never told it was difficult ;)